Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -390,6 +425,10 @@ Updates attributes of an existing Active Directory contact.
 #### Base Command
 
 `ad-update-contact`
+
+##### Required Permissions
+Requires `Create, delete, and manage user accounts` permissions.
```

Comment:
same here

Reply:
I didnt find anything about this specific permission but `Create, delete, and manage user accounts` should be enough